ompi/mpiext/ftmpi/mpif-h/comm_ishrink_f.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2010-2022 The University of Tennessee and the University
+ *                         of Tennessee Research Foundation.  All rights
+ *                         reserved.
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ */
+#include "ompi_config.h"
+
+#include "ompi/mpi/fortran/mpif-h/bindings.h"
+#include "ompi/mpi/fortran/base/constants.h"
+
+#include "ompi/mpiext/ftmpi/c/mpiext_ftmpi_c.h"
+#include "ompi/mpiext/ftmpi/mpif-h/prototypes_mpi.h"
+
+#include "ompi/communicator/communicator.h"
+#include "ompi/request/request.h"
+#include "ompi/mpi/fortran/base/fint_2_int.h"
+
+#if OPAL_HAVE_WEAK_SYMBOLS
+#pragma weak PMPIX_COMM_SHRINK = ompix_comm_shrink_f
```

Comment:
copy/paste issue. use ishrink

Reply:
Thanks for pointing out, there was actually more things wrong with the Fortran bindings, it was missing a number of f08 functions etc. It's all good now.